.devcontainer.json

```diff
@@ -7,7 +7,7 @@
     "esbenp.prettier-vscode",
     "timonwong.shellcheck"
   ],
-  "postCreateCommand": "scripts/initialise_devContainer.sh",
+  "postStartCommand": "scripts/initialise_devContainer.sh",
```

Comment:
Hi Ayman, I think we are going to have a clash between our 2 PRs. I need a mechanism to have a script to run once (and only once) at start-up - that is postCreateCommand. You need something that is run everytime to refresh the npm package .. . . 

So, can you add a new script in this PR
```
"postStartCommand": "scripts/install_packages.sh"
```
And then leave the existing line there? I will then be able to have my stuff unaffected by your PR?

Reply:
Obviously you will have to copy the current contents of initialise_devContainer into this new file

Reply:
Could we rename them to ?
devContainer_OnInitialise
devContainer_OnStart

Reply:
Not too fussed on what the script is called (I normally like the name to describe what it does rather than when it is called - but not precious on this)
As long as we end up leaving the postCreateCommand line - I'm happy

Reply:
I've reverted the change back and added a new script file devContainer_OnStart.sh to be called on postStartcommand event.